TECHMANIA/Assets/Scripts/Components/Main Menu/SelectTrackPanel.cs

```diff
@@ -507,6 +507,7 @@ private void OnTrackFilterChanged()
         errorTrackList.Clear();
 
         BuildTrackListFor(Paths.GetTrackRootFolder());
+        BuildTrackListFor(Paths.GetSaTrackRootFolder());
```

Comment:
This will throw DirectoryNotFoundException if the streaming assets folder doesn't exist.